tests/integration/defs/kv_cache/test_prefix_aware_scheduling.py

```
@@ -0,0 +1,799 @@
+
+                if server_log and _check_server_errors(server_log):
+                    return _kill("Server error detected in log, killing benchmark")
+
+                try:
+                    # 5s (was 2s) — under high offered load the async /health
+                    # endpoint can be briefly starved by the busy event loop.
+                    resp = req_lib.get(f"http://localhost:{port}/health", timeout=5)
+                    if resp.status_code == 200:
+                        last_health_ok = now
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)